### DISCUSSION TITLE
How to get user's phone number?

### DISCUSSION THREAD

**Discussion by yutanov** [python-telegram-bot/python-telegram-bot#2963] [Q&A] [answered]
Is there any way to get user's phone number?

**Comment by Bibo-Joshi** [comment, marked as answer]
https://github.com/python-telegram-bot/python-telegram-bot/wiki/Code-snippets#requesting-location-and-contact-from-user

**Comment by beaurty87** [comment]
But when I received the Excel file from Himalist, I realized – this was different. The names, companies, locations, and even verified phone numbers, along with emails, were all very clear.  Please visit our website: https://listtodata.com

**Comment by mubarakkhatri901-cell** [comment]
@Akhandyts

**Comment by mubarakkhatri901-cell** [comment]
P mujhe iska number chahie

**Comment by sentthilkumarlingesvaran-source** [comment]
Username to numper

**Reply by permanentfakeid-hub** [reply]
Haan mujhe bhi chahiye but free aur unlimited mere paas bot hai usme me 2 searches free hai mujhe api chahiye
